source/client-side-encryption/tests/legacy/fle2v2-CreateCollection.yml

```diff
@@ -19,7 +18,6 @@ tests:
         encryptedFieldsMap:
           default.encryptedCollection: &encrypted_fields0 {
                 "escCollection": "enxcol_.encryptedCollection.esc",
-                "eccCollection": "enxcol_.encryptedCollection.ecc",
```

Comment:
I noticed these `createCollection` tests never assert that `encryptedFieldsMap` is passed to the `create` command. That seems vital to the implementation of the `createCollection()` helper, so shouldn't we be doing that?

This also relates to my earlier comment in the spec file about removing the metadata collections from `encryptedFieldsMap` entirely, so we can then assert that the driver never sends values for these fields.

Reply:
The `state collections and index are created` test expects to get `encryptedFields` from the `encryptedFieldsMap`. The test asserts the `encryptedFields` is sent in the `create` command here: https://github.com/mongodb/specifications/blob/364464f97872d6894a80f07f43320adcbe8e29bf/source/client-side-encryption/tests/legacy/fle2v2-CreateCollection.yml#L117

Reply:
I definitely missed that. Thanks for pointing it out.